### PR TITLE
Yaswanth/88385/pnl calculator console warning fix

### PR DESCRIPTION
**Pull request by yaswanth-deriv**
Changes:

-   ...

## Type of change

-   [X] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1:

File: src/pages/dmt5/index.tsx [hunk 1]
```diff
@@ -46,7 +48,22 @@ const query = graphql`
         }
     }
 `
-
+const target = '#gatsby-focus-wrapper'
+const options = {
```
Comment (habib-deriv) [inline src/pages/dmt5/index.tsx:52]: Can we get this variable from constants, as this is getting re-used

File: src/pages/trade-types/cfds/index.tsx [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
 import { localize, WithIntl } from 'components/localization'
+import { useHandleLazyLoad } from 'components/hooks/use-handle-lazy-load'
+const WhatIsCFD = Loadable(() => import('./_what_is_cfd'))
 const WhyTradeCFD = Loadable(() => import('./_why-trade-cfd'))
 const TradingCFDIncreases = Loadable(() => import('./_trading-cfd-increases'))
 const StartTrading = Loadable(() => import('./_start-trading'))
 const ThingsToKeep = Loadable(() => import('./_mind-when-trading'))
 const AvailableMarkets = Loadable(() => import('./_available-markets'))
+const target = '#hero-section'
+const options = {
```
Comment (habib-deriv) [inline src/pages/trade-types/cfds/index.tsx:16]: This as well

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**